src/main/java/ladder/model/GameResultList.java

```diff
@@ -0,0 +1,36 @@
+package ladder.model;
+
+import ladder.model.utils.StringToListConverter;
+
+import java.util.List;
+
+public class GameResultList {
+    private static final int MAX_LENGTH = 5;
+    private static List<String> resultList;
```

Comment:
static일 필요는 없어 보이는데요, 제거하면 어떨까요?
```suggestion
    private final List<String> resultList;
```